aad/spring-cloud-azure-starter-active-directory/aad-resource-server-obo/pom.xml

```diff
@@ -11,7 +11,7 @@
 
   <artifactId>spring-cloud-azure-starter-active-directory-resource-server-obo</artifactId>
   <version>1.0.0</version>
-  <name>Azure Spring Boot Azure AD Starter Sample - Resource Server OBO</name>
+  <name>Spring-Cloud-Azure-Starter-Active-Directory Sample: Resource Server with On-Behalf-of Feature</name>
```

Comment:
`with` -> `With`.

Reply:
I changed all with to lowercase `with`;
since there are 11 display names has "with", I think it's better to keep them the same.